arangod/Cluster/ClusterInfo.h

```diff
@@ -412,6 +412,8 @@ class ClusterInfo final {
 
     std::unordered_map<ServerID, RebootId> rebootIds() const;
 
+    RebootId getCurrentRebootId(ServerID const& serverId) const;
+
```

Comment:
I think we should remove that message again. IMO the *only* place which may get rebootIds from other servers from the plan is the RebootTracker; all other places must only get it with the corresponding request.